Add 60ghz Wireless stuff

**Is your feature request related to a problem? Please describe.**
Add the 60ghz wireless stuff

**Describe the solution you'd like**
Add the commands for:
AP + Client -> /interface w60g
AP -> /interface w60g station

**Documentation**
https://help.mikrotik.com/docs/spaces/ROS/pages/39059501/W60G

The main interface under "/interface w60g" is always there, an default interface. On AP and client the same.

On the AP clients can be predefined under "/interface w60g station" but also can be created by routeros as an client connect with the correct settings

**Additional context**
I of couse can test it. And Merry Christmas!

Sorry, took me some time to find time for this.

resource "routeros_interface_w60g" "test" {
  name     = "wlan60-1"
  password = "put_your_safe_password_here"
  ssid     = "put_your_new_ssid_here"
  disabled = false
  mode     = "ap-bridge"
}

Error: POST 'https://172.31.10.31:443/rest/interface/w60g/set' returned response code: 400, message: 'Bad Request', details: 'missing =.id='
│
│   with routeros_interface_w60g.test,
│   on main.tf line 40, in resource "routeros_interface_w60g" "test":
│   40: resource "routeros_interface_w60g" "test" {

routeros_interface_w60g.test: Creating...
2025-05-13T09:49:00.552+0200 [INFO]  Starting apply for routeros_interface_w60g.test
2025-05-13T09:49:00.552+0200 [DEBUG] routeros_interface_w60g.test: applying the planned Create change
2025-05-13T09:49:00.553+0200 [DEBUG] provider.terraform-provider-routeros_v1.84.0.exe: request body:  {"disabled":"no","mode":"ap-bridge","name":"wlan60-1","password":"put_your_safe_password_here","ssid":"put_your_new_ssid_here"}: @module=routeros tf_provider_addr=terraform-routeros/routeros tf_req_id=872a2946-0b48-683a-17cf-2a2bcd372c11 tf_rpc=Configure @caller=github.com/terraform-routeros/terraform-provider-routeros/routeros/log.go:27 timestamp="2025-05-13T09:49:00.553+0200"
2025-05-13T09:49:00.553+0200 [DEBUG] provider.terraform-provider-routeros_v1.84.0.exe: POST request URL:  https://172.31.10.31:443/rest/interface/w60g/set: tf_provider_addr=terraform-routeros/routeros tf_req_id=872a2946-0b48-683a-17cf-2a2bcd372c11 tf_rpc=Configure @caller=github.com/terraform-routeros/terraform-provider-routeros/routeros/log.go:27 @module=routeros timestamp="2025-05-13T09:49:00.553+0200"
2025-05-13T09:49:00.564+0200 [DEBUG] provider.terraform-provider-routeros_v1.84.0.exe: error response body:
{"detail":"missing =.id=","error":400,"message":"Bad Request"}: tf_req_id=872a2946-0b48-683a-17cf-2a2bcd372c11 tf_rpc=Configure @caller=github.com/terraform-routeros/terraform-provider-routeros/routeros/log.go:27 @module=routeros tf_provider_addr=terraform-routeros/routeros timestamp="2025-05-13T09:49:00.564+0200"
2025-05-13T09:49:00.564+0200 [ERROR] provider.terraform-provider-routeros_v1.84.0.exe: Response contains error diagnostic: diagnostic_severity=ERROR tf_proto_version=5.8 tf_provider_addr=terraform-routeros/routeros tf_req_id=d18881d8-8f9c-8529-61cd-70640ef1f4f9 tf_resource_type=routeros_interface_w60g @caller=github.com/hashicorp/terraform-plugin-go@v0.26.0/tfprotov5/internal/diag/diagnostics.go:58 @module=sdk.proto diagnostic_detail="" diagnostic_summary="POST 'https://172.31.10.31:443/rest/interface/w60g/set' returned response code: 400, message: 'Bad Request', details: 'missing =.id='" tf_rpc=ApplyResourceChange timestamp="2025-05-13T09:49:00.564+0200"
2025-05-13T09:49:00.565+0200 [DEBUG] State storage *remote.State declined to persist a state snapshot
2025-05-13T09:49:00.565+0200 [ERROR] vertex "routeros_interface_w60g.test" error: POST 'https://172.31.10.31:443/rest/interface/w60g/set' returned response code: 400, message: 'Bad Request', details: 'missing =.id='

--- a/routeros/resource_interface_w60g.go
+++ b/routeros/resource_interface_w60g.go
@@ -131,9 +131,10 @@ func ResourceInterfaceW60g() *schema.Resource {
 	}
 
 	return &schema.Resource{
-		CreateContext: DefaultSystemCreate(resSchema),
-		ReadContext:   DefaultSystemRead(resSchema),
-		UpdateContext: DefaultSystemUpdate(resSchema),
+
+		CreateContext: DefaultCreateUpdate(resSchema),
+		ReadContext:   DefaultRead(resSchema),
+		UpdateContext: DefaultCreateUpdate(resSchema),
 		DeleteContext: DefaultSystemDelete(resSchema),
 
 		Importer: &schema.ResourceImporter{

